whitepapers/0007_governor.md

```diff
@@ -69,7 +74,7 @@ Each Guardian publishes its Governor configuration and status on the Wormhole go
 
 ## Detailed Design
 
-The Governor is implemented as an additional package that defines (1) a `ChainGovernor` object, (2) `mainnet_tokens.go`, a single map of tokens that will be monitored, and (3) `mainnet_chains.go`, a map of chains governed by the chain governor.
+The Governor is implemented as an additional package that defines (1) a `ChainGovernor` object, (2) `mainnet_tokens.go`, a single map of tokens that will be monitored, (3) `mainnet_chains.go`, a map of chains governed by the chain governor, and (4) `flow_cancel_tokens.go`, a map of tokens that can reduce the Governor's rate limit.
```

Comment:
>can reduce the Governor's rate limit.
nit: "can reduce a chains calculated aggregate flow." or some term that is more accurate than "reduce the ... limit" since the limit remains the same.